admin/client/EditorColorScaleSection.tsx

```diff
@@ -136,29 +152,48 @@ class ColorsSection extends React.Component<{
         const { scale } = this.props
 
         if (selected.value === "custom") {
-            scale.config.customColorsActive = true
+            scale.config.customNumericColorsActive = true
         } else {
             scale.config.baseColorScheme = selected.value
-            scale.config.customColorsActive = undefined
+            scale.config.customNumericColorsActive = undefined
         }
     }
 
     @action.bound onInvert(invert: boolean) {
         this.props.scale.config.colorSchemeInvert = invert || undefined
     }
 
-    @action.bound onAutomatic(isAutomatic: boolean) {
-        this.props.scale.config.isManualBuckets = isAutomatic ? undefined : true
+    @action.bound onBinningStrategy(
+        binningStrategy: ValueType<{
+            label: string
+            value: ColorScaleBinningStrategy
+        }>
+    ) {
+        this.props.scale.config.binningStrategy = asArray(
+            binningStrategy
+        )[0].value
     }
 
     @computed get currentColorScheme() {
         const { scale } = this.props
-        return scale.isCustomColors ? "custom" : scale.baseColorScheme
+        return scale.customNumericColorsActive
+            ? "custom"
+            : scale.baseColorScheme
     }
 
     render() {
         const { scale } = this.props
 
+        const binningStrategyOptions = entries(
```

Comment:
Does it make sense to generate this object on every render? Maybe better as a `computed`?

Reply:
Thanks! fixed now